src/Symfony/Component/Yaml/Inline.php

```diff
@@ -28,6 +28,7 @@ class Inline
     private static $exceptionOnInvalidType = false;
     private static $objectSupport = false;
     private static $objectForMap = false;
+    private static $constantSupport = false;
```

Comment:
My vote is for a flag too because this component should be as close to the Yaml spec as possible. `!php/const:` is a Symfony thing, not a Yaml thing.